Gemfile.lock

```diff
@@ -2,15 +2,23 @@ PATH
   remote: .
   specs:
     settings_yaml (1.0.0)
-      activesupport-core-ext (~> 4.0)
+      activesupport (~> 4.0)
```

Comment:
As this gem is intended to be used with Rails, it should be no problem. Although I wonder why it didn't work for you.